### PR TITLE
amam/reasset_index

### PR DESCRIPTION
**Pull request by cakasuma**
add new explanations for asset index

### CODE REVIEW

**Comment by negarn** [comment]
@cakasuma travis failing, please check

File: scripts/config/pages.js [hunk 1]
```diff
@@ -23,7 +23,8 @@ module.exports = [
     ['new_account/welcome',                     'app/new_account/welcome_page',                'default',    'Welcome to Binary.com'],
 
     ['resources',                               'app/resources/index',                         'default',    'Resources'],
-    ['resources/asset_indexws',                 'app/resources/asset_index',                   'full_width', 'Asset Index'],
+    ['resources/asset_indexws',                'app/resources/asset_index',                   'full_width', 'Asset Index'],
+    ['resources/trading_duration',              'app/resources/trading_duration',              'full_width', 'Trading Duration'],
```
Comment (negarn) [inline scripts/config/pages.js]: is it trading duration or trading durations?

File: src/templates/app/_layout/header.jsx [hunk 2]
```diff
@@ -116,6 +116,7 @@ const Header = () => (
                                             className: 'ico-only-hide client_logged_in nav-dropdown-toggle invisible',
                                             subitems : [
                                                 { text: it.L('Asset Index'),       href: it.url_for('resources/asset_indexws') },
+                                                { text: it.L('Trading Duration'),  href: it.url_for('resources/trading_duration') },
```
Comment (negarn) [inline src/templates/app/_layout/header.jsx]: Is it trading duration or trading durations?

File: src/templates/app/resources/trading_duration.jsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+                        <ArrowsMobile parent='duration_assets' direction='right' />
+                    </div>
+                    <TabsSubtabs
+                        id='duration_assets'
+                        className='gr-parent tab-selector-wrapper'
+                        items={[
+                            { id: 'tab_forex',       text: it.L('Forex'),               className: 'invisible' },
+                            { id: 'tab_indices',     text: it.L('Indices'),             className: 'invisible' },
+                            { id: 'tab_commodities', text: it.L('Commodities'),         className: 'invisible' },
+                            { id: 'tab_volidx',      text: it.L('Volatility Indices'),  className: 'invisible' },
```
Comment (negarn) [inline src/templates/app/resources/trading_duration.jsx]: do we need to hardcode market names?

File: src/templates/app/resources/trading_duration.jsx [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+                            { id: 'duration_assets_selector',                           className: 'tab-selector' },
+                        ]}
+                    />
+                    <div className='tab-content gr-padding-20'>
+                        <TabContentContainer>
+                            <TabContent id='tab_forex' className='selectedTab' />
+                            <TabContent id='tab_indices' />
+                            <TabContent id='tab_commodities' />
+                            <TabContent id='tab_volidx' />
```
Comment (negarn) [inline src/templates/app/resources/trading_duration.jsx]: same here

File: scripts/config/pages.js (hunk 1 above)
Comment (cakasuma) [reply scripts/config/pages.js]: it's trading durations for plural 👍

File: src/templates/app/resources/trading_duration.jsx (hunk 3 above)
Comment (cakasuma) [reply src/templates/app/resources/trading_duration.jsx]: i believe this is the only way to do it inside tab container 🙏

**Pull request closed without merging**